source/manual/incorrect-postcode-data.html.md

```diff
@@ -0,0 +1,21 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Incorrect postcode data
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+---
+
+We use postcode data from the ONS Postcode Directory (https://geoportal.statistics.gov.uk/search?q=ONS%20Postcode%20Directory%20(ONSPD) and Ordnance Survey for boundary
+line data. We typically [import the data into Mapit](https://github.com/alphagov/mapit/blob/master/IMPORTING-DATA.md) every 6 months.
+
+We sometimes receive Zendesk tickets about postcodes not returning the right results.
+You can check the accuracy of the postcode by:
+
+* Looking up the postcode on Mapit https://mapit.mysociety.org/postcode/<postcode>
```

Comment:
For some reason the `<postcode>` part of the URL disappears when the markdown render.

<img width="593" alt="mapit-url" src="https://user-images.githubusercontent.com/5963488/128001079-63d65a93-4ac1-43b8-8a56-3bc4ed6dfdef.png">

Maybe change to: 

```
Looking up the postcode on Mapit, for example https://mapit.mysociety.org/postcode/E18QS and having a look at the parents.
```